test/ProgressOnderwijsUtils.Tests/HtmlSanitizerTest.cs

```diff
@@ -104,6 +104,34 @@ public void RemoveObfuscatedJsLinks()
         public void HttpHrefsAreAllowed()
             => PAssert.That(() => HtmlFragment.ParseFragment(@"This is <a href=""https://somelink.com/?saynomore=1"">a link</a>.").Sanitize().ToStringWithoutDoctype() == @"This is <a href=""https://somelink.com/?saynomore=1"">a link</a>.");
 
+        [Fact]
+        public void MailtoHrefsAreAllowed()
+            => PAssert.That(() => HtmlFragment.ParseFragment(@"This is <a href=""mailto:bla@example.com"">a link</a>.").Sanitize().ToStringWithoutDoctype() == @"This is <a href=""mailto:bla@example.com"">a link</a>.");
+
+        [Fact]
+        public void MailtoHrefsCantHaveFragments()
+            => PAssert.That(() => HtmlFragment.ParseFragment(@"This is <a href=""mailto:bla@example.com#unsupported"">a link</a>.").Sanitize().ToStringWithoutDoctype() == @"This is a link.");
+
+        [Fact]
+        public void NoImgMailTos()
+            => PAssert.That(() => HtmlFragment.ParseFragment(@"This is <img src=""mailto:bla@example.com"">").Sanitize().ToStringWithoutDoctype() == @"This is ");
+
+        [Fact]
+        public void MailtoHrefsCantHavePaths()
+            => PAssert.That(() => HtmlFragment.ParseFragment(@"This is <a href=""mailto:bla@example.com/lala"">a link</a>.").Sanitize().ToStringWithoutDoctype() == @"This is a link.");
+
+        [Fact]
+        public void MailtoHrefsHaveSingleTo()
+            => PAssert.That(() => HtmlFragment.ParseFragment(@"This is <a href=""mailto:bla@example.com,foo@bar.com"">a link</a>.").Sanitize().ToStringWithoutDoctype() == @"This is a link.");
+
+        [Fact]
+        public void MailtoHrefsCanHaveSubjectAndBody()
+            => PAssert.That(() => HtmlFragment.ParseFragment(@"This is <a href=""mailto:foo@bar.com?sUbJect=asfdasdf&bODY=asdf"">a link</a>.").Sanitize().ToStringWithoutDoctype() == @"This is <a href=""mailto:foo@bar.com?sUbJect=asfdasdf&amp;bODY=asdf"">a link</a>.");

```

Comment:
Hoe werkt die sanitize voor de body dan? Kan je daarin ook weer een link of javascript opnemen?

Reply:
Is dit een goed idee? Dit is wel op zijn minst licht tricky want je kan hiermee xss elevaten omdat je nu html in een email kan injecten die een email client wellicht laadt. Zoals een img tag die xss doet via een get request. Ik zou ervoor zijn dit niet toe te staan eigenlijk

Reply:
Dit is op basis van wat googlen; mijn conclusie was dat een totaal free-form mailto voor de meeste clients veilig is, maar als er al risicos zijn, dat het beperkt is tot de extra argumenten.  Ik kon nergens iets vinden dat een mail client er impliciet html van maakt ofzo.

Laten we wel wezen, 't is 2021 zo'n mail client bestaat waarschijnlijk gewoon niet.

Reply:
(immers was zo'n client al decennia lang super onveilig geweest)